lib/ProductOpener/Import.pm

```diff
@@ -412,6 +412,167 @@ my %no = (
 );
 
 
+sub apply_organization_settings_to_product($$$$) {
```

Comment:
I think there is a naming issue, as what the code below does is to check if the source is authorized to load data for the organization.

The apply_organization_settings_to_product is good name, but for the part of the code that was below that check, starting after # If it is a GS1 import (Equadis, CodeOnline, Agena3000), check if the org is associated with known issues